InternshipApp/InternshipApp/Screens/ProfileScreenView/ProfileScreenView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ProfileScreenView.swift
+//  InternshipApp
+//
+//  Created by Sead Mašetić on 13. 8. 2024..
+//
+
+import SwiftUI
+
+struct ProfileScreenView: View {
+    
+    @StateObject var profileViewModel: ProfileScreenViewModel
+    @StateObject var taskViewModel: TaskViewModel
+    
+    var body: some View {
+        NavigationStack {
+            VStack {
+                if(profileViewModel.image != nil) {
```

Comment:
no need to use "()" when writing if statements in swift :) - not wrong but not commonly used